src/main/scala/mesosphere/marathon/storage/repository/Repositories.scala

```diff
@@ -71,7 +72,7 @@ object GroupRepository {
     ctx: ExecutionContext,
     mat: Materializer): StoredGroupRepositoryImpl[RamId, String, Identity] = {
     import mesosphere.marathon.storage.store.InMemoryStoreSerialization._
-    new StoredGroupRepositoryImpl(store, appRepository, podRepository)
+    new StoredGroupRepositoryImpl(store, appRepository, podRepository, 1000)
```

Comment:
Where is this magic number coming from?

Reply:
it was a default parameter in the GroupRepositoryImpl before:
![image](https://user-images.githubusercontent.com/1043441/41239189-1a8822dc-6d98-11e8-9400-3f90f622082b.png)

Reply:
You should pass `versionCacheMaxSize` here too